frontend/src/js/components/editors/template/TemplateLabelSettings.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+
+var TemplateLabelSettings = React.createClass({
+    propTypes: {
+        label: React.PropTypes.object,
+        update: React.PropTypes.func
+    },
+    updateLabelValue(key) {
+        const self = this;
+        let validator = parseFloat;
```

Comment:
What is validator? is it a function? `e.tartget.value` if less than 0 then equal 0; if greater than 1 then equal 1;